modal_proto/api.proto

```diff
@@ -618,10 +612,8 @@ message QueueCreateResponse {
 
 message QueueGetRequest {
   string queue_id = 1;
-  bool block = 2 [deprecated=true];
   float timeout = 3;
   int32 n_values = 4;
-  string idempotency_key = 5 [deprecated=true];
```

Comment:
We should either keep it or add some kind of reservation dummy for the field id since it's the last one on the request and might get reused otherwise

Reply:
yeah i thought about that, but this field hasn't been used for an extremely long time so I think it's fine?